js/okcoinusd.js

```diff
@@ -272,6 +274,41 @@ module.exports = class okcoinusd extends Exchange {
         let ticker = this.extend (response['ticker'], { 'timestamp': timestamp });
         return this.parseTicker (ticker, market);
     }
+    
+    parseTickers (tickers, timestamp, symbols) {
+        if (!tickers || (tickers.length < 1)) {
+            return {};
+        }
+        let tickers_result = {};
+        for (let i = 0; i < tickers.length; i++) {
+            let market = undefined;
+            if ('symbol' in tickers[i]) {
+                if (tickers[i]['symbol'] in this.markets_by_id) {
+                    market = this.markets_by_id[tickers[i]['symbol']];
+                }
+            }
+            let ticker = this.extend (tickers[i], { 'timestamp': timestamp });
+            tickers_result[market['symbol']] = this.parseTicker (ticker, market);
+        }
+        if (!symbols || (symbols.length < 1)) {
```

Comment:
The same type of length-clause here, need a separate variable for the symbols count.